dotnet/src/webdriver/Internal/Base64UrlEncoder.cs

```diff
@@ -23,7 +23,7 @@ namespace OpenQA.Selenium.Internal
     /// <summary>
     /// Encodes and Decodes strings as Base64Url encoding.
     /// </summary>
-    internal static class Base64UrlEncoder
+    public static class Base64UrlEncoder
```

Comment:
I don't like making it public, will redo it via `InteralsVisibleTo` test assembly.